receiver/statsdreceiver/receiver.go

```diff
@@ -117,7 +117,7 @@ func (r *statsdReceiver) Start(ctx context.Context, host component.Host) error {
 						r.Flush(ctx, metrics, r.nextConsumer)
 					}
 				case rawMetric := <-transferChan:
-					r.parser.Aggregate(rawMetric)
+					r.parser.Aggregate(rawMetric, r.config.EnableMetricType)
```

Comment:
For this part, it looks wired to parse the config flag to aggregate each time we receive the metric. The config setup needs to be done before each metric comes. For example, we could have a `getEnableMetricTypeFlag` function to do the setup.

Reply:
Good point, it makes sense to make this value initialized in the config setup part, I will change the code.